sandbox/src/mocks/stu3/acceptReferral/responses/ExampleResponse.json

```diff
@@ -249,6 +249,23 @@
                     "value": "103264992985"
                   }
                 }
+              },
+              {
+                "url": "https://fhir.nhs.uk/STU3/StructureDefinition/Extension-eRS-AvailabilityStatus-1",
+                "valueCoding":
+                {
+                  "system": "https://fhir.nhs.uk/STU3/CodeSystem/eRS-AvailabilityStatus-1",
+                  "code": "AVAILABLE"
+                }
+              },
+              {
+                "url": "https://fhir.nhs.uk/STU3/StructureDefinition/Extension-eRS-AttachedByOrganisation-1",
+                "valueReference": {
+                  "identifier": {
+                    "system": "https://fhir.nhs.uk/Id/ods-organization-code",
+                    "value": "R69"
+                  }
```

Comment:
Should more of these examples have the AttachedOnBehalfOf extension? The story seems to say so